protocol/avdl/chat1/common.avdl

```diff
@@ -187,6 +187,12 @@ protocol common {
     bytes k; // Verifying key
   }
 
+  record SignEncryptedData {
+    int   v; // version = 1
+    bytes b; // signEncryptedData (output of signencrypt.SealWhole)
+    bytes n; // nonce
```

Comment:
(from a now-hidden comment)

I put the verify key up top instead of including it in SignEncryptedInfo but could be talked into doing that instead easily. It seems sort of good to me for the program to have to explicitly get the verification key out to facilitate remembering to validate that it belongs to the correct person.

Also, it may or may not be a good idea to duplicate it inside the header. To protect against non-exclusive ownership problems. But I'm not sure about that?